src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -220,4 +227,7 @@ def tearDown(self):
             try:
                 server_utils.stop_server(hosts=self.hostlist_servers)
             finally:
+                #Storage reset
+                if self.nvme_parameter == "nvme":
+                    server_utils.storage_reset(','.join(self.hostlist_servers))
```

Comment:
This should be in a try block just in case the `clustershell_execute` function raises a `ServerFailed` exception.  We would still want `super().tearDown()` to be called in this case.